packages/trader/src/AppV2/Components/TradeTypeList/trade-type-list-item.tsx

```diff
@@ -1,35 +1,53 @@
 import React from 'react';
 import { StandaloneCirclePlusFillIcon } from '@deriv/quill-icons';
+import clsx from 'clsx';
 
 type TTradeTypeListItemProps = {
     title: string;
+    selected: boolean;
     leftIcon?: React.ReactNode;
     rightIcon?: React.ReactNode;
     onLeftIconClick?: () => void;
     onRightIconClick?: () => void;
+    onTradeTypeClick?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
 };
 
 const TradeTypeListItem: React.FC<TTradeTypeListItemProps> = ({
     title,
+    selected,
     leftIcon,
     rightIcon,
     onLeftIconClick,
     onRightIconClick,
+    onTradeTypeClick,
 }) => {
     const default_icon = <StandaloneCirclePlusFillIcon fill='var(--core-color-solid-green-700)' iconSize='sm' />;
 
     return (
-        <div className='trade-type-list-item'>
+        <button
+            className={clsx('trade-type-list-item', { 'trade-type-list-item--selected': selected })}
+            onClick={onTradeTypeClick}
+        >
             {leftIcon && (
-                <button className='trade-type-list-item__left-icon' onClick={onLeftIconClick}>
+                <button
+                    className='trade-type-list-item__left-icon'
+                    data-testid='dt_trade_type_list_item_left_icon'
+                    onClick={onLeftIconClick}
+                >
                     {leftIcon}
                 </button>
             )}
             <div className='trade-type-list-item__title'>{title}</div>
-            <button className='trade-type-list-item__icon' onClick={onRightIconClick}>
-                {rightIcon || default_icon}
-            </button>
-        </div>
+            {onRightIconClick && (
```

Comment:
Does it matter that here we are checking if `onRightIconClick` was passed and in case of leftIcon on line 31 we are checking the presence of `leftIcon` ?